pageObject/inventoryPage.js

```diff
@@ -31,8 +66,45 @@ class InventoryPage extends Page {
     return $(".shopping_cart_link");
   }
 
-  open() {
-    super.open("inventory.html");
+  get sorter() {
+    return $(".product_sort_container");
+  }
+
+  async sortedArray(cat) {
+
+    const alphAscSorted = await this.itemTitles.map((e) => e.getText()).sort();
+    const arrayOfPrices = await this.itemPrices
```

Comment:
You have a method here is it suite? https://github.com/milabakhmat/webdriverio/pull/3/files#diff-0c7df4866c37a67f1617531ec480176725d3751df8aa665ce3ad1a2cff8aa258R27